ansible/roles_ocp_workloads/ocp4_workload_portworx/files/requirements.yml

```diff
@@ -1,22 +1,22 @@
 ---
```

Comment:
Is this file actually used? Seems like it belongs into the `ocp4-cluster` config (where it already is with newer dependencies).

Reply:
I removed the file